spectral/tests/database.py

```diff
@@ -23,33 +23,42 @@
 ECOSTRESS_DATA_DIR = os.path.join(os.path.split(__file__)[0],
                                   'data/ecostress')
 ECOSTRESS_DB = os.path.join(testdir, 'ecostress.db')
+USGS_DATA_DIR = os.path.join(os.path.split(__file__)[0],
+                             'data/usgs/ASCIIdata')
+USGS_DB = os.path.join(testdir, 'usgs.db')
 AVIRIS_BAND_FILE = os.path.join(os.path.split(__file__)[0], 'data/92AV3C.spc')
 
+
 class ECOSTRESSDatabaseCreationTest(SpyTest):
-    '''Tests that SpyFile memmap interfaces read and write properly.'''
+    '''Tests ECOSTRESS database creation from text files.'''
+
     def __init__(self):
         pass
 
     def setup(self):
         if not os.path.isdir(testdir):
             os.makedirs(testdir)
+        if os.path.exists(ECOSTRESS_DB):
+            os.remove(ECOSTRESS_DB)
```

Comment:
Useful if we want to run tests multiple times, or if we develop and debug by writing tests.